File: readability/readability.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>
#include <ctype.h>

int count_letters(string text);

int main(void)
{
    string s = get_string("Text: ");
    printf("Letters: %i\n", count_letters(s));
}

int count_letters(string text)
{
    int count = 0;
    for (int i = 0; i < strlen(text); i++)
    {
        if (isalpha(text[i]) != 0)
        {
            count++;
        }
    }
    return count;
}

File: sentimental-readability/readability.py
from cs50 import get_string

def main():
    string = get_string("Text: ")
    L = 100.0 * count_letters(string) / count_words(string)
    S = 100.0 * count_sentences(string) / count_words(string)
    index = 0.0588 * L - 0.296 * S - 15.8; # Coleman-Liau Formula

    if index < 1.0:
        printf("Before Grade 1\n")

    elif index > 16.0:
        printf("Grade 16+\n")

    elif 1.0 < index < 16.0:
        printf("Grade %i\n", rounded_index)

def count_letters(string):
    num_letters = 0
    for i in range(0, len(string)) # check if letter
    {
        if string[i].isalpha()
        {
            num_letters++;
        }
    }
    return num_letters;

main()